cirq/google/common_serializers.py

```diff
@@ -280,3 +280,24 @@ def _near_mod_2(e, t, atol=1e-8):
         gate_constructor=lambda: ops.FSimGate(theta=-np.pi / 4, phi=0),
         args=[]),
 ]
+"""Measurement serializer."""
```

Comment:
Fix comments.  Also, Matthew suggested in a different PR that we use comments instead of string literals to improve formatting (I'll fix the other ones)

Reply:
Done.